### ISSUE TITLE
Changing graph filter above  6 hours blanks out graph lines

### ISSUE THREAD

**Issue by mtdeguzis** [Icinga/icingaweb2-module-graphite#227]
## Expected Behavior
Even if my storage retention is only 6 hours, the right side of the graph should still have the data.

## Current Behavior
6 hours link, debug text, and image.
![image](https://user-images.githubusercontent.com/27829577/71207795-a7cd7780-2275-11ea-91f9-8e9be26e2ea3.png)

```
# Link
https://udaicinga.domain.com:8484/icingaweb2/monitoring/service/show?host=host.domain.com&service=check-ping4&graph_range=21600&graph_debug=1

# Debug
+ Icinga check command: 'ping4'
+ Obscured check command: NULL
+ Applying templates for check command 'ping4'
++ Applying template 'ping-rta'
+++ Fetched 1 metric(s) from 'http://icinga-prod-graphite-01:443/metrics/expand?query=icinga2.host_domain_conf.services.check-ping4.ping4.perfdata.rta.value'
+++ Excluded 0 metric(s)
+++ Combined 1 metric(s) to 1 chart(s)
++ Applying template 'ping-pl'
+++ Fetched 1 metric(s) from 'http://icinga-prod-graphite-01:443/metrics/expand?query=icinga2.host_domain_conf.services.check-ping4.ping4.perfdata.pl.value'
+++ Excluded 0 metric(s)
+++ Combined 1 metric(s) to 1 chart(s)
+ Applying default templates, excluding previously used metrics
++ Not applying template 'default-host'
++ Applying template 'default-service'
+++ Fetched 2 metric(s) from 'http://icinga-prod-graphite-01:443/metrics/expand?query=icinga2.host_domain_conf.services.check-ping4.ping4.perfdata.%2A.value'
+++ Excluded 2 metric(s)
+++ Combined 0 metric(s) to 0 chart(s)
```

12 hours link, debug text, and image.
![image](https://user-images.githubusercontent.com/27829577/71207815-b2880c80-2275-11ea-849d-4aaa92477554.png)

```
# Link
https://udaicinga.domain.com:8484/icingaweb2/monitoring/service/show?host=host.domain.cp,&service=check-ping4&graph_debug=1&graph_range=43200

# Debug
+ Icinga check command: 'ping4'
+ Obscured check command: NULL
+ Applying templates for check command 'ping4'
++ Applying template 'ping-rta'
+++ Fetched 1 metric(s) from 'http://icinga-prod-graphite-01:443/metrics/expand?query=icinga2.host_domain_conf.services.check-ping4.ping4.perfdata.rta.value'
+++ Excluded 0 metric(s)
+++ Combined 1 metric(s) to 1 chart(s)
++ Applying template 'ping-pl'
+++ Fetched 1 metric(s) from 'http://icinga-prod-graphite-01:443/metrics/expand?query=icinga2.host_domain_com.services.check-ping4.ping4.perfdata.pl.value'
+++ Excluded 0 metric(s)
+++ Combined 1 metric(s) to 1 chart(s)
+ Applying default templates, excluding previously used metrics
++ Not applying template 'default-host'
++ Applying template 'default-service'
+++ Fetched 2 metric(s) from 'http://icinga-prod-graphite-01:443/metrics/expand?query=icinga2.host_domain_com.services.check-ping4.ping4.perfdata.%2A.value'
+++ Excluded 2 metric(s)
+++ Combined 0 metric(s) to 0 chart(s)
```

## Your Environment

* Module version (System - About): 1.1.0
* Icinga Web 2 version and modules (System - About): 2.7.3
* Icinga 2 version (`icinga2 --version`): 2.11.2-1
* Graphite (carbon, web, whisper) version: 
* Operating System and version: RHEL 7. 
* Webserver, PHP versions: 
```
# /opt/rh/rh-php71/root/usr/sbin/php-fpm -v
PHP 7.1.30 (fpm-fcgi) (built: Jul 10 2019 09:46:42)
```

**Comment by mtdeguzis** [comment]
I haven't really changed anything retention settings-wise, but wanted to also confirm the files I use:

[conf.zip](https://github.com/Icinga/icingaweb2-module-graphite/files/4225266/conf.zip)

**Comment by axelhahn** [comment]
Hi,

If I had the issue: in the graphite webinterface I verified it [https://[graphite-server]/dashboard/] - here was the same behavior like in the icingaweb2: the older data in icinga2.* were also not available here. In my case it was a problem of Graphite.

Inside the *wsp files Graphite stores the retention settings if the first data element was sent. If the retention (in storage.schemas.conf) will be changed it won't be applied on existing datasets. Solution:

(1)
There is a python tool whisper-resize.py to apply new retention settings on a single wsp file. 
Or 
(2)
You delete the wsp files (this is what I did - it is painless if you have no stored older data so far). The first newly sent data by Icinga will set the wanted retention.

HTH

Regards,
Axel
